CONTRIBUTING.md

```diff
@@ -49,6 +49,23 @@ make docs-in-container
 ```
 and then documentation can be find in */docs/build/html*.
 
+
+#### Changelog
+
+When you are contributing to changelog, please follow these suggestions:
+
+* The changelog is meant to be read by everyone. Imagine that an average user
+  will read it and should understand the changes. `Add check timeouts` is
+  completely undescriptive.
```

Comment:
Hey, I do not want to be a counter-example..;)

Reply:
k, I'll change it so that I'm the example :P